smarts/env/gymnasium/wrappers/metric/metrics.py

```diff
@@ -237,8 +234,22 @@ def reset(self, **kwargs):
                     raise MetricsError(
                         "Unsupported configuration for distance-to-destination cost function."
                     )
+                start_pos = Point(*self._scen.missions[agent_name].start.position)
```

Comment:
```suggestion
                start_pos = self._scen.missions[agent_name].start.point
```

Reply:
I had missed the in-built `point` method. Now changed to use the `point` method.